charts/admission-controller/values.yaml

```diff
@@ -71,6 +73,12 @@ sysdig:
   # Overrides the `global.sysdig.secureAPIToken` parameter.
   secureAPIToken: ""
 
+  # **required for KSPM AC**
+  # <br/>Access Key to access Sysdig Secure.
+  # <br/><br/>Either this value or `sysdig.existingAccessKeySecret` is required
+  # Overrides `global.sysdig.accessKey`
+  accessKey: ""
```

Comment:
Since `sysdig-deploy` already supports it, can add also the `accessKeySecret` and its usage?

Reply:
That's in our backlog - SSPROD-27044

The change is not completely trivial. If you consider this mandatory for this release (this will be Controlled Availability) I can prioritize that bug ticket. Otherwise, I would release without it and the team will handle the implementation.

Reply:
Just thinking about those users that already have the `accessKeySecret`. If they want to enable this `kspmAdmissionController` they'll be forced to add `accessKey` as well 😅

Reply:
You are right, I'll try to expedite this